linebot/response.go

```diff
@@ -76,6 +76,68 @@ type MessageConsumptionResponse struct {
 	TotalUsage int64
 }
 
+// MessagesNumberDeliveryResponse type
+type MessagesNumberDeliveryResponse struct {
+	Status          string `json:"status"`
+	Broadcast       int64  `json:"broadcast"`
+	Targeting       int64  `json:"targeting"`
+	AutoResponse    int64  `json:"autoResponse"`
+	WelcomeResponse int64  `json:"welcomeResponse"`
+	Chat            int64  `json:"chat"`
+	APIBroadcast    int64  `json:"apiBroadcast"`
+	APIPush         int64  `json:"apiPush"`
+	APIMulticast    int64  `json:"apiMulticast"`
+	APIReply        int64  `json:"apiReply"`
+}
+
+// MessagesNumberFollowersResponse type
+type MessagesNumberFollowersResponse struct {
+	Status          string `json:"status"`
+	Followers       int64  `json:"followers"`
+	TargetedReaches int64  `json:"targetedReaches"`
+	Blocks          int64  `json:"blocks"`
+}
+
+// MessagesFriendDemographicsResponse type
+type MessagesFriendDemographicsResponse struct {
+	Available           bool                       `json:"available"`
+	Genders             []GenderDetail             `json:"genders"`
+	Ages                []AgeDetail                `json:"ages"`
+	Areas               []AreaArrayDetail          `json:"areas"`
+	AppTypes            []AppTypeDetail            `json:"appTypes"`
+	SubscriptionPeriods []SubscriptionPeriodDetail `json:"subscriptionPeriods"`
+}
+
+// GenderDetail type
+type GenderDetail struct {
+	Gender     string  `json:"gender"`
+	Percentage float64 `json:"percentage"`
+}
+
+// AgeDetail type
+type AgeDetail struct {
+	Age        string  `json:"Age"`
+	Percentage float64 `json:"percentage"`
+}
+
+// AreaArrayDetail type
+type AreaArrayDetail struct {
```

Comment:
To follow other struct names such as `AgeDetail`,  `GenderDetail` and others, could this be named `AreaDetail` instead of `AreaArrayDetail`?

Reply:
`AreaDetail` already in use by richmenu.go https://github.com/line/line-bot-sdk-go/blob/4a254ebfebf5a6c8a152b9cd3e8de3e8be6ad6c3/linebot/richmenu.go#L64

Any naming suggestion?

Reply:
How about Area**s**Detail?
I think this is not goot but better than AreaArrayDetail.

Reply:
I already fixed it, let me know if you have any question.